controllers/storagecluster/volumesnapshotterclasses.go

```diff
@@ -130,6 +130,11 @@ func (r *StorageClusterReconciler) createSnapshotClasses(vsccs []SnapshotClassCo
 
 // ensureCreated functions ensures that snpashotter classes are created
 func (obj *ocsSnapshotClass) ensureCreated(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) (reconcile.Result, error) {
+
+	if IsOCSConsumerMode(instance) {
+		return reconcile.Result{}, nil
+	}
```

Comment:
I think you need a similar guard in `ensureDeleted` below (Line 149)

Reply:
added